ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -402,8 +416,12 @@ class FamilyReads extends React.PureComponent {
     const rnaTrackOptions = RNA_TRACK_TYPE_OPTIONS.filter(({ value }) => igvSampleIndividuals[value])
     const project = openFamily && this.getProjectForFamily(openFamily)
     const geneLocus = project && variant && getGeneLocus(variant, genesById, project)
-    const locusOptions = geneLocus ? [{ value: getVariantLocus(variant, project), text: 'Variant' },
+    const geneLocusOptions = geneLocus ? [{ value: getVariantLocus(variant, project), text: 'Variant' },
       { value: geneLocus, text: 'Gene' }] : null
+    const spliceOutlierLocus = variant && getSpliceOutlierLocus({ ...variant, familyGuids: [openFamily] },
+      spliceOutliersByChromFamily[variant.chrom])
+    const locusOptions = spliceOutlierLocus ?
+      (geneLocusOptions || []).concat([{ value: spliceOutlierLocus, text: 'Splice Outlier' }]) : geneLocusOptions
```

Comment:
I think this is a bit more convoluted than it needs to be, rather than merging 2 potentially null list you can just do a direct assignment for each element and then filter
```
const locusOptions = [
  { text: 'Variant', value: project && variant && getVariantLocus(...) },
  { text: 'Gene' , value: project && variant && getGeneLocus(...) },
  { text: 'Splice Outlier', value: variant && getSpliceOutlierLocus(...) }
].filter(({ value }) => value)
```